website/blog/2023-06-11-databend-weekly-97.md

```diff
@@ -36,7 +36,7 @@ Stay connected with the latest news about Databend.
 
 ### Column Position
```

Comment:
I am not sure if I understand this feature correctly. Please review this part carefully....

Reply:
@PsiACE